module/modmanager/manager.go

```diff
@@ -1089,11 +1090,12 @@ func (mgr *Manager) FirstRun(ctx context.Context, conf config.Module) error {
 	firstRunPath, markSuccess, err := conf.EvaluateFirstRunPath(packages.LocalPackagesDir(mgr.packagesDir))
 	if err != nil {
 		// TODO(RSDK-9067): some first run path evaluation errors should be promoted to WARN logs.
-		logger.Debug("no first run script detected, skipping setup phase", "error", err)
+		logger.Debugw("no first run script detected, skipping setup phase", "error", err)
 		return nil
 	}
 
-	logger.Info("executing first run script")
+	logger = logger.With("path", firstRunPath)
```

Comment:
I love including an identifier like this so someone can grep for all of the logs pertaining to a single run.